### ISSUE TITLE
Mismatch in yaml keys

### ISSUE THREAD

**Issue by Yash151** [carbonblack/cbapi-python#236]
For the file present in the location:
https://github.com/carbonblack/cbapi-python/blob/master/src/cbapi/response/models/process.yaml

For the **group** key there are two type sub keys one of which should be a **format**. So the change should be from:

```
  group:
    type: string
    **type**: string
```

To;

```
  group:
    type: string
    **format**: string
```

**Comment by avanbrunt-cb** [comment]
Fix pushed to master to remove additional type as format is not used for string https://github.com/carbonblack/cbapi-python/commit/73325f2879cfcd8c6a4d7d6a4881bfb0cee157d5

**Comment by avanbrunt-cb** [comment]
This will be released with 1.7.0
